modules/auxiliary/scanner/mssql/mssql_schemadump.rb

```diff
@@ -27,18 +25,18 @@ def initialize
 
     register_options([
       OptBool.new('DISPLAY_RESULTS', [true, "Display the Results to the Screen", true])
-      ])
+    ])
   end
 
   def run_host(ip)
 
     if !mssql_login_datastore
-      print_error("#{rhost}:#{rport} - Invalid SQL Server credentials")
+      print_error("#{datastore['RHOST']}:#{datastore['RPORT']} - Invalid SQL Server credentials")
       return
     end
 
     # Grabs the Instance Name and Version of MSSQL(2k,2k5,2k8)
-    instancename = mssql_query(mssql_enumerate_servername())[:rows][0][0].split('\\')[1]
+    instancename = mssql_query(mssql_enumerate_servername())[:rows][0][0].split('\\')[0]
```

Comment:
this a bug you found/fixed?

Reply:
I believe so - see https://github.com/rapid7/metasploit-framework/pull/18696#discussion_r1471965136